syntax = "proto2";

message get_device_infoRequest {}

message get_device_infoResponse {
    required string type = 1; 
    required string product = 2;
    required string ip = 3;
}

message claimRequest {}

message claimResponse {
    required string token = 1;
}

message reclaimRequest {
    required string token = 1;
}

message unclaimRequest {
    required string token = 1;
}

message set_smartledRequest {
    required string token = 1;
    required int32 id = 2;
    required uint32 color = 3;
}